x/stake/client/cli/tx.go

```diff
@@ -87,6 +87,16 @@ func GetCmdCreateValidator(cdc *codec.Codec) *cobra.Command {
 				)
 			}
 
+			if viper.GetBool(FlagGenesisFormat) {
```

Comment:
I think you can combine this with the below conditional block.

Reply:
Done, please have a look. I tried to minimise the indentation level.